addons/calendar/models/discuss_channel_member.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+class DiscussChannelMember(models.Model):
+    _inherit = 'discuss.channel.member'
+
+    def _rtc_invite_members(self, member_ids=None):
+        if self.channel_id.calendar_event_id and fields.Datetime.now() < self.channel_id.calendar_event_id.start:
+            return
```

Comment:
`_rtc_invite_members` is a rather low level function for inviting, I don't think we should prevent that one.

It would be best to prevent the call to `_rtc_invite_members` from `_rtc_join_call` by adding an override point for the existing condition.